src/os/bluestore/bluestore_types.h

```diff
@@ -156,7 +156,8 @@ struct denc_traits<vector<bluestore_pextent_t>> {
   static void bound_encode(const vector<bluestore_pextent_t>& v, size_t& p) {
     p += sizeof(uint32_t);
     size_t per = 0;
-    denc(*(bluestore_pextent_t*)nullptr, per);
+    bluestore_pextent_t tmp;
```

Comment:
If it's faulting out here, then either:

(1) the denc code that determines the length of bluestore_pextent_t is incorrect. This code must NOT reference the object. It's job is to determine the worst-case length that the encoded object would take. That determination is made independent of the data itself. Thus that code shouldn't be examining the referenced object (which is *properly* null).

(2) A compiler bug.